api-schema/src/main/java/org/stellar/anchor/api/callback/PutCustomerRequest.java

```diff
@@ -67,6 +67,12 @@ public class PutCustomerRequest {
   @SerializedName("clabe_number")
   String clabeNumber;
 
+  @SerializedName("cbu_number")
+  String cbuNumber;
+
+  @SerializedName("cbu_alias")
+  String cbuAlias;
```

Comment:
Did I forget to add these fields in the 1.2.4 release? I see changes here I thought we already merged to develop?

Reply:
The PR is now rebased to the `develop` branch to include these fields.